JamGame/Assets/Scripts/Employee/EmployeeModifiers.cs

```diff
@@ -130,7 +130,6 @@ public void RegisterBuff(Buff buff)
             }
         }
 
-        // TODO: match type of effect with corresponding Executor type.
```

Comment:
Calling side working with `IEffect`, so type information must be recovered this way anywhere